src/main/java/jenkins/plugins/logstash/persistence/BuildData.java

```diff
@@ -67,15 +67,20 @@
   private final static Logger LOGGER = Logger.getLogger(MethodHandles.lookup().lookupClass().getCanonicalName());
   public static class TestData {
     private int totalCount, skipCount, failCount, passCount;
-    private List<FailedTest> failedTestsWithErrorDetail;
+    private List<ExecutedTest> failedTestsWithDetail;
```

Comment:
This will change the JSON format in an incompatible way
The name has to stay the same for sake of backwards compatibility

Reply:
Oh, sorry, that was obvious and I totally misread your first comment. I'll flip that back and remove the extra classes.